doc/classes/Font.xml

```diff
@@ -279,7 +279,7 @@
 				To print available variation axes of a variable font:
 				[codeblock]
 				var fv = FontVariation.new()
-				fv.set_base_font = load("res://RobotoFlex.ttf")
+				fv.set_base_font(load("res://RobotoFlex.ttf"))
```

Comment:
```suggestion
				fv.base_font = load("res://RobotoFlex.ttf")
```

Reply:
Thanks for the feedback. I've amended this patch. I will also push another patch updating the `FontVariation` documentation, which uses `set_base_font()`.